packages/fast-development-site-react/src/components/site/action-bar.tsx

```diff
@@ -83,6 +83,8 @@ function menuButtonBase(): ICSSRules<IDevSiteDesignSystem> {
 const styles: ComponentStyles<IActionBarClassNameContract, IDevSiteDesignSystem> = {
     actionBar: {
         display: "flex",
+        flexBasis: "100%",
+        borderBottom: `${toPx(1)} solid #CCCCCC`
```

Comment:
Not a design system color, or cannot be made using Chroma?

Reply:
@nalogiudice I'm not following - what are you asking for?

Reply:
Take a look at checkbox - it uses Chroma.mix to achieve grays using the design system colors. I'm wondering if we should be doing that everywhere